tests/mocks/proxy_implementation.cairo

```diff
@@ -0,0 +1,70 @@
+%lang starknet
```

Comment:
maybe name this file something like `proxiable_implementation.cairo`? it took me a moment to understand if it was supposed to be a kind of proxy.

Reply:
Good call. I can see how this can be ambiguous